src/pipecat/processors/frame_processor.py

```diff
@@ -296,11 +296,11 @@ async def __cancel(self, frame: CancelFrame):
         await self.__cancel_push_task()
 
     async def __pause(self, frame: FrameProcessorPauseFrame | FrameProcessorPauseUrgentFrame):
-        if frame.name == self.name:
+        if frame.processor.name == self.name:
```

Comment:
it would actually be `frame.processor`. Maybe the filed should be `process_name` to avoid confusion.

Reply:
I just updated the PR. Let me know if it makes sense.

Reply:
I agree. I just avoided using `FrameProcessor` for the circular imports. But you can actually avoid that with `TYPE_CHECKING`.